.github/workflows/tests.jsonnet

```diff
@@ -220,6 +221,29 @@ local install_x86_artifacts = {
   |||,
 };
 
+local download_x86_pro_artifacts = {
+  uses: 'actions/download-artifact@v3',
+  with: {
+    name: core_pro_x86.export.artifact_name,
+  },
+};
+local install_x86_pro_artifacts = {
+  name: 'Install pro artifacts',
+  run: |||
+    tar xf ocaml-build-artifacts.tgz
+
+    # All binaries will be placed in cli/tmp-bin first
+    # before copying over to /usr/bin. This is so that
+    # we don't need to run semgrep with sudo.
+    mv ocaml-build-artifacts/bin cli/tmp-bin
+    cd cli
+    PATH="tmp-bin:$PATH" pipenv run semgrep install-semgrep-pro --custom-binary tmp-bin/semgrep-core-proprietary
```

Comment:
Can you update the PR description and specify what kind of issues you want to find and what issues you will not find this way?

Reply:
Done.